controllers/v1/sprint.go

```diff
@@ -241,3 +268,15 @@ func (ctrl SprintController) GetSprintMemberSummary(c *gin.Context) {
 
 	c.JSON(status, response)
 }
+
+// GetTrails is method to get the all trails related to a particular sprint
+func (ctrl SprintController) GetTrails(c *gin.Context) {
```

Comment:
Add a permission check for the sprint in the API, it should be accessible only to the users who can access the sprint.

Reply:
Done